mysite/_config/config.yml

```diff
@@ -1,21 +0,0 @@
----
-Name: mysiteconfig
-After:
-  - 'framework/*'
-  - 'cms/*'
----
-# YAML configuration for SilverStripe
-# See http://doc.silverstripe.org/framework/en/topics/configuration
-# Caution: Indentation through two spaces, not tabs
-SiteTree:
-  extensions:
-    - WorkflowEmbargoExpiryExtension
-SSViewer:
-  theme: 'starter'
-MySQLDatabase:
-  connection_charset: 'utf8'
-LeftAndMain:
-  help_link: "https://www.cwp.govt.nz/user-help/"
-VersionFeed:
-  allchanges_enabled: false
-  changes_enabled: false
```

Comment:
Where did this end up going?

Reply:
`silverstripe/versionfeed` is no longer installed by this recipe - do you think it should be kept, so having it off by default seems no longer necessary.

Reply:
Ok so it _could_ go into the recipe that now contains versionfeed, but I assume that these default are different to the defaults from the module (otherwise they wouldn't be here) so they need to stay here rather than being moved to the new recipe where they'd always override the module defaults. CWP specific config can stay here.

Reply:
I did check, and they are [both `true` by default](https://github.com/silverstripe/silverstripe-versionfeed/blob/master/src/VersionFeed.php#L38).
I'm assuming here that because it used to be a part of the megalithic recipe (but not always desired) these are here to simply [_disable_ the functionality](https://github.com/silverstripe/silverstripe-versionfeed/blob/master/docs/en/developer.md#enabling--disabling) by default, in said old recipe.

Reply:
Right, but since we'll still be bundling that module with the CWP installer by default it'd make sense to keep these config options?

Reply:
Added back in the new `versionfeed.yml`

Reply:
In case you were wondering, it's a massive "attack" vector for a lot of sites that don't even use versionfeed actively - bots like to crawl the whole site with `/changes` or `/allchanges` and it breaks almost every site that lets it. Hence the PR I made a while ago to disable it by default :)